tests/shared/utils/test_common.py

```diff
@@ -175,17 +174,7 @@ def test_class_from_module_path_not_found(
         rasa.shared.utils.common.class_from_module_path(module_path, lookup_path)
 
 
-@pytest.mark.parametrize(
-    "module_path, result, outcome",
-    [
-        ("rasa.shared.core.domain.Domain", rasa.shared.core.domain.Domain, True),
-        ("rasa.shared.core.domain.logger", rasa.shared.core.domain.logger, False),
-    ],
-)
-def test_class_from_module_path_ensure_class(
-    module_path: Text, outcome: bool, result: Any, recwarn: WarningsRecorder
-):
-    klass = rasa.shared.utils.common.class_from_module_path(module_path)
-    assert klass is result
-
-    assert bool(len(recwarn)) is not outcome
+def test_class_from_module_path_fails():
```

Comment:
I think we also need a test for happy path?

Reply:
Happy path is already tested here:
https://github.com/RasaHQ/rasa/blob/bd4ca3f62377eab72264c5421916bcd1c2091fa0/tests/shared/utils/test_common.py#L146-L159

Reply:
👍 I should have looked in the file before asking 😄